regexes/device/mobiles.yml

```diff
@@ -1221,6 +1221,72 @@ Dicam:
   device: 'feature phone'
   model: '$1'
 
+# Digma
+Digma:
+  regex: 'Digma[_ ]([^;/]+) Build|HIT HT707[10]MG|CITI 1902 3G|(CITI (?:[A-Z0-9]+) 3G) C[ST]500[67]PG|iDrQ10 3G|iDxD8 3G|iDnD7|HIT 4G HT7074ML|IDX5|(iDx10|iDx7)|MVM900H(?:WZ|C)|MVM908HCZ|iDxD10 3G|(iDxD[45]|iDxQ5)|iDxD7[_ ]3G|PS604M|PT452E|Linx A400 3G LT4001PG|Linx C500 3G LT5001PG|Linx PS474S|NS6902QL|NS9797MG|((?:Optima|Platina|Plane)[ _](?:[EM])?(?:(?:[0-9\.ST]+|Prime))(?:[ _][43]G)?)|(VOX[ _](?:[0-9\.A-Z]+)[_ ][43]G)'
+  device: 'smartphone'
+  models:
+    - regex: 'HIT HT707[10]MG'
+      model: 'HIT 3G'
+      device: 'tablet'
+    - regex: '(IDX5)'
+      model: '$1'
```

Comment:
It's better to avoid reges where they aren't needed. In this case it would be better/enough to write:
```
- regex: 'IDX5'
  model: 'IDX5'
```
This can be changed for some regex below as well